code/modules/mob/living/carbon/xenomorph/items/iff_tag.dm

```diff
@@ -0,0 +1,66 @@
+/obj/item/iff_tag
+	name = "xenomorph IFF tag"
+	desc = "A tag containing a small IFF computer that gets inserted into the carapace of a xenomorph. You can modify the IFF groups by using an access tuner on it, or on the xeno if it's already implanted."
+	icon = 'icons/obj/items/xeno_tag.dmi'
+	icon_state = "xeno_tag"
+	var/list/faction_groups = list()
+
+/obj/item/iff_tag/attack(mob/living/carbon/Xenomorph/xeno, mob/living/carbon/human/injector)
+	if(isXeno(xeno))
+		injector.visible_message(SPAN_NOTICE("[injector] starts forcing \the [src] into [xeno]'s carapace..."), SPAN_NOTICE("You start forcing \the [src] into [xeno]'s carapace..."))
+		if(!do_after(injector, 5 SECONDS, INTERRUPT_ALL, BUSY_ICON_GENERIC, xeno, INTERRUPT_DIFF_LOC, BUSY_ICON_GENERIC))
+			return
+		injector.visible_message(SPAN_NOTICE("[injector] forces \the [src] into [xeno]'s carapace!"), SPAN_NOTICE("You force \the [src] into [xeno]'s carapace!"))
+		xeno.iff_tag = src
+		injector.drop_inv_item_to_loc(src, xeno)
+		return
+	return ..()
```

Comment:
should not be able to put more than one tag into a xeno as it causes large issues + should also not be able to put them in dead xenos